gems/pending/util/win32/miq-powershell.rb

```diff
@@ -221,9 +225,13 @@ def process_array(node)
     def process_hash(node)
       hsh = {}
       node.each_element do |e|
-        name = convert_type(e.elements[1])
+        # REXML starts its elements indexing at 1, Nokogiri at 0.
```

Comment:
Does this take into account XmlHash?

@gmcculloug Can you verify that XmlHash would work properly here...or do the tests cover it?